app/src/main/java/com/mobiledevelopmentfoodapp/dto/Customer.kt

```diff
@@ -1,10 +1,12 @@
 package com.mobiledevelopmentfoodapp.dto
 
-//import androidx.room.Entity
-//import androidx.room.PrimaryKey
 import com.google.gson.annotations.SerializedName
 
-data class Customer(var name: String, var phone: String, @SerializedName("id") var customerId:Int = 0 ) {
+data class Customer(
+    internal var name: String,
+    internal var phone: String,
+    @SerializedName("id")internal var customerId: Int = 0,
```

Comment:
Added trailing comma. A trailing comma is a stylistic choice. The Kotlin style guide *recommends* a trailing comma. Best practices for trailing commas can be found here. https://kotlinlang.org/docs/coding-conventions.html#trailing-commas